### DISCUSSION TITLE
Google Analytics 4 integration in cordova android

### DISCUSSION THREAD

**Discussion by ayethinzarkhine** [apache/cordova#379] [Q&A]
Anyone can use Google Analytics 4 in cordova android? I am trying to migrate UA to GA4 but GA4 properties not receiving data at all. I am trying to connect through GTM or gtag.js both not working.

**Comment by breautek** [comment]
I don't think the "web" versions of analytic frameworks works as they are tied to a domain name and I believe the google servers verifies by attempting to connect to the "website" which in a cordova context, doesn't actually exists and is not reachable.

You'll need to use the native version of the analytics which are tied to the bundle or app id. Cordova doesn't maintain any official plugin for this purpose, but there are [third-party plugins](https://www.npmjs.com/search?q=ecosystem%3Acordova%20analytics) available.

**Comment by keriroso** [comment]
Use this plugin https://github.com/chemerisuk/cordova-plugin-firebase-analytics it's work for me.
